.changeset/lemon-phones-sort.md

```diff
@@ -0,0 +1,37 @@
+---
+"@vercel/edge-config": major
+---
+
+ - **BREAKING CHANGE** Return values are now read-only to improve in-memory caching
+
+   It used to be possible to change the returned value as seen this example:
+
+   ```typescript
+    import { get } from '@vercel/edge-config';
+    const countries = await get('allowedCountryCodes');
+    countries.DE = true; // Won't be possible anymore and throw an error
```

Comment:
This will not actually throw anymore now that we removed the proxy, right?

Reply:
Yes, updated it now